plugins/kotlin-dataframe/testData/box/playground.kt

```diff
@@ -0,0 +1,9 @@
+import org.jetbrains.kotlinx.dataframe.*
+import org.jetbrains.kotlinx.dataframe.annotations.*
+import org.jetbrains.kotlinx.dataframe.api.*
+import org.jetbrains.kotlinx.dataframe.io.*
+
+fun box(): String {
```

Comment:
this is just so you don't have to create this file every time you need to quickly test something? ;P

Reply:
Yes indeed :) You need to create a file and run generateTests, it will modify BlackBoxTestsGenerated and you'd also need to rollback it if it was only a temporary test. Very inconvenient :c